pulsar-client-admin/pom.xml

```diff
@@ -107,6 +107,12 @@
       <artifactId>pulsar-package-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Interesting. Why do we need this dependency explicitly now?

Reply:
I don't know why ~, maybe the old testNG includes this library?